pandas/src/inference.pyx

```diff
@@ -1256,23 +1270,25 @@ def fast_multiget(dict mapping, ndarray keys, default=np.nan):
     return maybe_convert_objects(output)
 
 
-def downcast_int64(ndarray[int64_t] arr, object na_values,
-                   bint use_unsigned=0):
+def downcast_int64(ndarray[int_dtype] arr, object na_values,
```

Comment:
Yep, no issues there.  It's pretty internal, so that can be changed without issue.